soweego/wikidata/sample_additional_info.py

```diff
@@ -0,0 +1,130 @@
+import json
+import csv
+import os
+import re
+from collections import defaultdict
+import click
+from ..target_selection.musicbrainz import common
+
+# Queries computing
+
+def query_info_for(qids_bucket, properties):
+    """Given a list of wikidata entities returns a query for getting some external ids"""
+
+    query = "SELECT * WHERE{ VALUES ?id { %s } " % ' '.join(qids_bucket)
+    for k, v in properties.items():
+        query += 'OPTIONAL { ?id wdt:%s %s . } ' % (v, k)
+    query += "}"
+    return query
+
+def query_wikipedia_articles_for(qids_bucket):
+    """Given a list of wikidata entities returns a query for getting wikidata articles"""
+
+    query = "SELECT * WHERE{ VALUES ?id { %s } " % ' '.join(qids_bucket)
```

Comment:
**Minor:** for more readabilty, please wrap the query in a triple quote string here, i.e., `"""QUERY"""`.